src/styles/card.scss

```diff
@@ -0,0 +1,98 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
+  background: #FFF;
+
+  &__image {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__name {
+    color: $main-accent;
+    font-size: $main-size;
+    font-weight: 500;
+    line-height: $main-line-height;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    color: $secondary;
+    font-size: $secondary-size;
+    font-weight: 400;
+    line-height: $secondary-line-height;
+    margin-bottom: 16px;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    margin-bottom: 24px;
+
+    &-text {
+      color: $main-accent;
+      font-size: $secondary-size;
+      line-height: $secondary-line-height;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    margin-bottom: 16px;
+
+    &-text {
+      font-size: $main-size;
+      color: $secondary;
+      font-style: normal;
+      font-weight: 400;
+      line-height: $main-line-height;
+    }
+
+    &-sum {
+      color: $main-accent;
+      font-size: 16px;
+      font-weight: 700;
+      line-height: $main-line-height;
+    }
+  }
+
+  &__buy-link {
+    text-decoration: none;
+  }
+
+  &__button {
+    text-decoration: none;
+    display: flex;
+    color: #fff;
+    align-items: center;
```

Comment:
Let's try to organize your CSS
I recommend you take a look at these styles code and use it in the feature
[9elements](https://9elements.com/css-rule-order) or [htmlAcademy](https://codeguide.academy/html-css.html#css-order)